src/sentry/api/helpers/group_index/update.py

```diff
@@ -738,3 +732,30 @@ def update_groups(
         result["inbox"] = inbox
 
     return Response(result)
+
+
+def handle_is_bookmarked(
+    is_bookmarked: Any,
```

Comment:
I believe this could be a bool if you call the function with `result["isBookmarked"]`.